src/Datadog.Trace.ClrProfiler.Managed/Emit/ObjectExtensions.cs

```diff
@@ -86,6 +89,11 @@ public static bool TryGetPropertyValue<TResult>(this object source, string prope
 
         public static MemberResult<TResult> GetProperty<TResult>(this object source, string propertyName)
         {
+            if (source == null)
```

Comment:
Same as [the previous comment](https://github.com/DataDog/dd-trace-dotnet/pull/427/files#r300093298). Consider throwing `ArgumentNullException`.

Reply:
Never mind. Discussed in previous comment.